packages/jupytercad-app/style/base.css

```diff
@@ -1,8 +1,9 @@
 #jp-top-panel {
   min-height: 60px !important;
   padding: 10px !important;
-  justify-content: flex-end;
-  padding-right: 40px !important;
+  justify-content: flex-start;
+  padding-left: 40px !important;
+  padding-right: 30px !important;
   border-bottom: none;
   background: #053b3e;
 }
```

Comment:
After this line, there are a couple of theme overwrite here:

```
:root {
  --jp-ui-font-color1: rgba(#4e5c65, 1) !important;
}
:root {
  --jp-inverse-layout-color3: #09777c !important;
}
:root {
  --jp-brand-color1: #09777c !important;
}
```

These don't play well with the dark theme

Reply:
I'm thinking should we reuse the dark/light theme of JupyterLab? The best way maybe is to create the dark and light theme for JupyterCad

Reply:
Probably having custom JupyterCad themes would be the way to go

Reply:
Indeed, we can do it in another PR